SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -187,9 +189,9 @@ + (YapDatabaseDeserializer)logOnFailureDeserializer
             OWSProdErrorWParams(@"storage_error_deserialization", ^{
                 return (@{
```

Comment:
Yup, it's the old tension between making the "reusable" flexible "enough" - able to handle the common use cases without making it crazy complicated to handle the edge cases.  Right now, there's only one place we need an extra param.